### PR TITLE
docs(types): fix type definitions

### LINKED ISSUE #1563: Docs -> Form -> Props -> Size property - Showing lodash 'without' function
### Steps
Go to Form documentation page. Look at Props table, size property.

### Expected Result
Show the all possible sizes.

### Actual Result
A lodash function: _.without(...)

image

### Version
0.67.2

I would love to work on this. It seems to affect another pages as well. Can I have a try? Someone can point me to the right direction?

Thanks.

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #1563.

### Before

image

### After

image

### CODE REVIEW

File: gulp/plugins/util/parseType.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}))
+
+const transformValues = (items) => _.flatMap(items, item => {
+  const { value } = item
+
+  if (_.startsWith(value, '...SUI')) return evalValue(value.substring(3))
+  return item
+})
+
+const parseEnum = type => {
```
Comment (levithomason) [inline gulp/plugins/util/parseType.js]: It looks like this reduces the need for some of the logic in `ComponentProps.js`:

[expandEnums()](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Components/ComponentDoc/ComponentProps.js#L116)
[renderEnums()](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Components/ComponentDoc/ComponentProps.js#L124)

**Review by layershifter — COMMENTED** [review]
I've updated parser functions. `renderEnums` method was simpilified, too :+1:

**Comment by levithomason** [comment]
Awesome, thanks much sir!  This will also help with the speed a bit, there's a pretty bad performance bug I need to trace down in the component doc page.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.4`.

**Pull request merged**